### ISSUE TITLE
tooltip encoding breaks line mark

### ISSUE THREAD

**Issue by nyurik** [vega/vega-lite#3220]
This graph works, but try changing mark to `"line"`, and everything disappears.  Also, try `"area"` -- there is some strange drawing artifacts.

```json
{
  "$schema": "https://vega.github.io/schema/vega-lite/v2.json",
  "data": {
    "url": "https://gist.githubusercontent.com/nyurik/daa888a0dfa7d1a3a15b2e57e2b1c4e4/raw/5c8904f85e2674bffb364c5049d6e5627ddb2172/logstashrandom.json",
    "format": {
      "property": "aggregations.time_buckets.buckets",
      "type": "json"
    }
  },
  "transform": [
    {
      "calculate": "if(datum.doc_count > 0, format(datum.doc_count, ',d') + ' documents', 'no documents')",
      "as": "tip"
    }
  ],
  "mark": "bar",
  "encoding": {
    "x": {
      "field": "key",
      "type": "temporal"
    },
    "y": {
      "field": "doc_count",
      "type": "quantitative"
    },
    "tooltip": {
      "field": "tip"
    }
  },
  "autosize": {
    "type": "fit",
    "contains": "padding"
  }
}
```

**Comment by domoritz** [comment]
Tooltip is adding another group by clause. I don't think there is anything we can do in Vega-Lite. The easiest solution for you is to add a second layer with point marks and only add tooltips to the point marks.

**Comment by domoritz** [comment]
Closing as there is no immediate action here.

**Comment by kanitw** [comment]
I disagree.  Since this is a raw plot, I think we have a bug in terms of determining line groupings

**Comment by kanitw** [comment]
Will be fixed in #3224
